docs/ha_3scale.adoc

```diff
@@ -0,0 +1,192 @@
+:toc:
+:toc-placement!:
+
+= 3scale HA different scenarios
+
+toc::[]
+
+== Common configuration
+
+* This section explains how to install 3scale using the link:https://github.com/3scale/3scale-operator[3scale Operator] and how to configure it to have a highly-available deployment
+* The first thing that you need to do is install the operator following  link:https://github.com/3scale/3scale-operator/blob/3scale-2.10-stable/doc/operator-user-guide.md#installing-3scale[this guide]
+* After installing the operator, you need to deploy an `APIManager` custom resource, that will be read by the operator and then deploy 3scale configured according to its spec
+* There is a basic example of an `APIManager` link:https://github.com/3scale/3scale-operator/blob/3scale-2.10-stable/doc/operator-user-guide.md#basic-installation[here]
+* In order to set up an HA deployment you need to tune a few things in the `APIManager` spec for every component
+
+=== External databases
+
+* You need to tell the Operator not to deploy the critical DBs following link:ha_dbs.adoc[this 3scale HA databases section] of this repo
+* Notice that the operator does not offer an option to deploy `system-memcached` and `system-sphinx` externally *because those two are not considered critical*, so you do not need an HA deployment of them
```

Comment:
```suggestion
* Notice that the operator does not offer an option to deploy `system-memcached` or `system-sphinx` externally. Neither of these components have a critical impact on 3scale in the event they are down, so the operator does not currently support a high-available deployment option for them.
```

Saying *you don't need* and HA deployment for them is a bit an extreme/strong statement.